source/common/filesystem/win32/watcher_impl.cc

```diff
@@ -194,7 +194,7 @@ void WatcherImpl::directoryChangeCompletion(DWORD err, DWORD num_bytes, LPOVERLA
       events |= Events::Modified;
     }
 
-    constexpr absl::string_view data{"a"};
+    char data[] = {'a'};
```

Comment:
why did you need to change this? string_view can be used as well.

Reply:
I changed it since keeping it would require a cast at line 208 to get rid of const, as the RawSlice constructor expects the first parameter to be void *, not const void *.  The char array is also smaller than string_view, though that hardly matters.

I guess there's no explicit need for this change though, so I'm happy to change it back to minimize number of lines churned.

Reply:
> The char array is also smaller than string_view, though that hardly matters.

While char array is smaller than string_view, a `constexpr` implies `static` so there's no allocation in runtime. I prefer keep this constexpr.

Reply:
Sure, I'll update.